File: Assignment/2학년 1학기/C++/0426/유저 캐릭터,아이템 관리_싱글톤/Human.cpp
#include "Human.h"

Human::Human(const char* _name) : Character(_name) {
	Character::SetHP(Character::GetHP() + 50);
	Character::SetMP(Character::GetMP() + 25);
	Character::SetAtt(Character::GetAtt() + 5);
	Character::SetDef(Character::GetDef() + 2);
	Character::SetBroodCode(HUMAN);
}

Human::~Human() {

}

File: Assignment/3학년 1학기/Java/기말고사/클래스 설계/Human.java
package Project1;

public class Human extends Character {
	
	public Human() {
		this.name = "None";
		this.brood = "인간";
	}
	public Human(String name) { 
		this.name = name;
		this.brood = "인간";
	}
	
	public void ActiveSkill()
	{
		System.out.println("협공");
	}
	
	public void PassiveSkill()
	{
		System.out.println("지력 상승");
	}
}
